duckduckgo/iOS
# Network Protection debug menu

Task/Issue URL: https://app.asana.com/0/0/1205084446087078/f
Tech Design URL:
CC:

**Description**:

Until now, the option to clear the auth token (thus resetting the invite flow) was in the status view. Now that we’re moving to an actual MVP, it makes sense to move this functionality to a debug menu item.

I give you: the `NetworkProtectionDebugViewController`

**Steps to test this PR**:
1. Go to Settings -> Network Protection
2. Make sure you’ve run the [invite code flow](https://app.asana.com/0/0/1205174820992574/f) so that the status view is showing
3. Go back to Settings
4. Go to Debug -> Network Protection
5. Tap the “Clear auth token” cell
6. Go back to the Network Protection screen (step 1)
7. **Check that the Invite Code flow shows again**

<!—
Before submitting a PR, please ensure you have tested the combinations you expect the reviewer to test, then delete configurations you *know* do not need explicit testing.

Using a simulator where a physical device is unavailable is acceptable.
—>

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
